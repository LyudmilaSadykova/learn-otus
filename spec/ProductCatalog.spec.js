import { mount } from '@vue/test-utils';
import component from '../src/components/ProductCatalog.vue';
import { createTestingPinia } from '@pinia/testing';

describe('ProductCatalog component', () => {
    const buildWrapper = () => mount(component, {
        global: {
            plugins: [createTestingPinia()],
        },
    });

    it('mount without errors', () => {
        const wrapper = buildWrapper();
        expect(wrapper.exists()).toBe(true);
        expect(wrapper.element).toMatchSnapshot();
    });
    
});
