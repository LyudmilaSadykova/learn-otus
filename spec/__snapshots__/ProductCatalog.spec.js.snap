// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`ProductCatalog component > mount without errors 1`] = `
<div
  data-v-app=""
>
  
  <h1
    data-v-18d94d88=""
  />
  <v-row
    data-v-18d94d88=""
    justify="center"
  >
    
    
  </v-row>
  
</div>
`;
